### ISSUE TITLE
Chromecast device gives up after trying to load

### ISSUE THREAD

**Issue by pawandubey** [keredson/gnomecast#60]
I tried casting to my chromecast device (built-in, Android TV) and it seems to succeed in finding the device and connecting to it but the streaming never starts (even when no transcoding is required). Running on a KDE Neon (based on Ubuntu 16.04).

```
... 4.13.0-38-generic #43~16.04.1-Ubuntu SMP Wed Mar 14 17:48:43 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```

Info about the video I was trying to stream (via `ffmpeg -i`):
```
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from '/home/pawan/Downloads/OT1080.mp4':
  Metadata:
    major_brand     : isom
    minor_version   : 512
    compatible_brands: isomiso2avc1mp41
    encoder         : Lavf54.63.104
  Duration: 00:27:49.73, start: 0.000000, bitrate: 2838 kb/s
    Stream #0:0(und): Video: h264 (High) (avc1 / 0x31637661), yuv420p, 1920x1080, 2670 kb/s, 29.97 fps, 29.97 tbr, 10000k tbn, 59.94 tbc (default)
    Metadata:
      handler_name    : VideoHandler
    Stream #0:1(und): Audio: aac (LC) (mp4a / 0x6134706D), 44100 Hz, stereo, fltp, 159 kb/s (default)
    Metadata:
      handler_name    : SoundHandler
```

**Comment by terminalcommand** [comment]
I am experiencing the same issue.  I've tried with several movies, powercyled my chromecast. 

My best guess is that a recent update broke it. It was working fine until now. 

The command output is:
  ```
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/gnomecast.py", line 21, in <module>
    import html5lib.treebuilders
ModuleNotFoundError: No module named 'html5lib'
No module named 'html5lib'
No subtitles. -1 None
None
serving on http://192.168.0.110:46825
check_ffmpeg
b'/usr/local/bin/ffmpeg\n'
Chromecast('192.168.0.100', port=8009, device=DeviceStatus(friendly_name='TardisTV', model_name='Chromecast', manufacturer='Google Inc.', uuid=UUID('eab9a830-cb86-c05c-fd3f-a220bda5c777'), cast_type='cast'))
Open clicked
File selected: /tmp/incorporateds01e01.mkv
Checking for subtitles... -1 None
Transcoder /tmp/incorporateds01e01.mkv mkv h264 True
True False True
['ffmpeg', '-i', '/tmp/incorporateds01e01.mkv', '-c:v', 'copy', '-c:a', 'mp3', '/tmp/gnomecast_9zmgw335.mp4']
subtitle_ids []
No subtitles. -1 None
CastStatus(is_active_input=False, is_stand_by=True, volume_level=1.0, volume_muted=False, app_id='CC1AD845', display_name='Default Media Receiver', namespaces=['urn:x-cast:com.google.cast.cac', 'urn:x-cast:com.google.cast.broadcast', 'urn:x-cast:com.google.cast.media'], session_id='2e5494c3-78c3-4add-b445-b351efc2837e', transport_id='2e5494c3-78c3-4add-b445-b351efc2837e', status_text='Default Media Receiver')
<MediaStatus {'metadata_type': None, 'title': None, 'series_title': None, 'season': None, 'episode': None, 'artist': None, 'album_name': None, 'album_artist': None, 'track': None, 'subtitle_tracks': {}, 'images': [], 'supports_pause': True, 'supports_seek': True, 'supports_stream_volume': True, 'supports_stream_mute': True, 'supports_skip_forward': False, 'supports_skip_backward': False, 'current_time': 0, 'content_id': 'http://192.168.0.110:41369/media/9096477244874987014.mkv', 'content_type': 'video/mp4', 'duration': None, 'stream_type': 'BUFFERED', 'idle_reason': 'ERROR', 'media_session_id': 1, 'playback_rate': 1, 'player_state': 'IDLE', 'supported_media_commands': 15, 'volume_level': 1, 'volume_muted': False, 'media_custom_data': {}, 'media_metadata': {}, 'current_subtitle_tracks': [], 'last_updated': datetime.datetime(2018, 6, 25, 12, 45, 16, 819924)}>
[('Content-Type', ''), ('Content-Length', '0'), ('Host', '192.168.0.110:46825'), ('Connection', 'keep-alive'), ('Accept-Encoding', 'identity;q=1, *;q=0'), ('User-Agent', 'Mozilla/5.0 (X11; Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.58 Safari/537.36 CrKey/1.29.104827'), ('Range', 'bytes=0-'), ('Chrome-Proxy', 'frfr'), ('Accept', '*/*'), ('Accept-Language', 'en-US,en;q=0.9'), ('Cast-Device-Capabilities', '{"bluetooth_supported":true,"display_supported":true,"hi_res_audio_supported":false}')]
self.transcoder.fn /tmp/gnomecast_9zmgw335.mp4
ranges [(0, 1000000000000)]
192.168.0.100 - - [25/Haz/2018:15:45:35 +0300] "GET /media/8703619115910319446.mkv HTTP/1.1" 404 750 "-" "Mozilla/5.0 (X11; Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.58 Safari/537.36 CrKey/1.29.104827"
```

**Comment by keredson** [comment]
@terminalcommand looks like you're missing `html5lib`.  can you install it and see if that fixes?

**Comment by keredson** [comment]
@pawandubey can you like to the file, and confirm what version of gnomecast you have?

**Comment by terminalcommand** [comment]
@keredson I've installed html5lib. Now there isn't any problem in video playback. 

I don't know whether the issue was related to html5lib.  

If anyone is experiencing this issue, I'd advise rebooting *both* the computer and the chromecast. 

Sorry for the inconvenience, when I saw the issue on github I thought the googlecast protocol had changed. I feel a bit stupid now, seeing that the solution was simply turning it off and on again :).

@keredson BTW: I've looked into the package on pip. Might I suggest adding html5lib to /gnomecast-1.4.0/gnomecast.egg-info/requires.txt . The requirements.txt here on github contains html5lib, but the pip package doesn't.  

Thanks for the fast reply
